core/src/main/scala/org/scalatra/util/MultiMapHeadView.scala

```diff
@@ -12,21 +10,42 @@ object MultiMapHeadView {
 
 }
 
-trait MultiMapHeadView[A, B] extends Map[A, B] {
+trait MultiMapHeadView[A, B] {
 
   protected def multiMap: Map[A, Seq[B]]
 
-  override def get(key: A): Option[B] = multiMap.get(key) flatMap { _.headOption }
+  def apply(key: A): B = multiMap.get(key) match {
+    case Some(v) => v.head
+    case None => throw new NoSuchElementException(s"Key ${key} not found")
+  }
+
+  def get(key: A): Option[B] = multiMap.get(key) flatMap { _.headOption }
+
+  def getOrElse(key: A, default: => B): B = toMap.get(key) getOrElse default
+
+  def size: Int = multiMap.size
 
-  override def size: Int = multiMap.size
+  def foreach(f: ((A, B)) ⇒ Unit): Unit = multiMap foreach { case (k, v) => f((k, v.head)) }
```

Comment:
If there is no strong reason, we should use `=>` instead of `⇒` for code style unification. I know that this kind of characters are used in some places of Scalatra codebase, and I think that we should replace them if we have a chance.

Reply:
I accept.

I copied it from the definition of the existing Map.

fix it